core/include/detray/propagator/aborters.hpp

```diff
@@ -0,0 +1,68 @@
+/** Detray library, part of the ACTS project (R&D line)
```

Comment:
I think you need to add this file into `CMakeLists.txt`

Reply:
Indeed, I forgot again :)